woudc_data_registry/epicentre/instrument.py

```diff
@@ -145,12 +156,14 @@ def add(ctx, station, dataset, name, model, serial, geometry):
     instrument_ = {
         'station_id': station,
         'dataset_id': dataset,
+        'contributor_id': contributor,
         'name': name,
         'model': model,
         'serial': serial,
-        'x': geom_tokens[1],
-        'y': geom_tokens[0],
-        'z': geom_tokens[2]
+        'start_date': datetime.now(),
+        'x': int(geom_tokens[1]),
+        'y': int(geom_tokens[0]),
+        'z': int(geom_tokens[2])
     }
```

Comment:
This whole dict also eventually gets passed to an `Instrument.__init__` too, so it also needs to have `contributor` and `project` as keys, not `contributor_id`.